.github/workflows/default.yaml

```
@@ -94,7 +94,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        os: [ ubuntu-22.04 ]
+        os: [ ubuntu-22.04, macos-latest ]
```

Currently, `macos-14` [is the same](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories) as `macos-latest`, but we should use the former for the same reasons we are currently using `ubuntu-22.04` rather than `ubuntu-latest`.

```suggestion
        os: [ ubuntu-22.04, macos-14 ]
```